### DISCUSSION TITLE
Federated credentials for S2S

### DISCUSSION THREAD

**Discussion by cegekaJG** [microsoft/AL-Go#917] [Q&A]
When following the guide to register a SaaS environment for continuous employment, I noticed that it's possible to register GitHub workflows as federated credentials. I don't know much about this feature, but would it allow actions like "Deploy to <environment>" to connect to an SaaS environment without specifying a client secret? Is this already supported in AL Go?

**Comment by freddydk** [comment]
I haven't seen this - could you send me a link to where you read this?

**Reply by cegekaJG** [reply]
Of course, it's [here](https://learn.microsoft.com/en-us/entra/identity-platform/quickstart-register-app#add-a-federated-credential).

**Comment by freddydk** [comment]
Federated credentials support was added last year.
